editor/src/components/canvas/design-panel-root.tsx

```diff
@@ -29,6 +35,72 @@ interface NumberSize {
 
 const TopMenuHeight = 40
 
+const NothingOpenCard = betterReactMemo('NothingOpen', () => {
+  const dispatch = useEditorState((store) => store.dispatch, 'NothingOpenCard dispatch')
+  const handleOpenCanvasClick = React.useCallback(() => {
+    dispatch([EditorActions.setPanelVisibility('canvas', true)])
+  }, [dispatch])
+  const handleOpenCodeEditorClick = React.useCallback(() => {
+    dispatch([EditorActions.setPanelVisibility('codeEditor', true)])
+  }, [dispatch])
+  const handleOpenBothCodeEditorAndDesignToolClick = React.useCallback(() => {
+    dispatch([
+      EditorActions.setPanelVisibility('codeEditor', true),
+      EditorActions.setPanelVisibility('canvas', true),
+    ])
+  }, [dispatch])
+
+  return (
+    <div
+      role='card'
+      style={{
+        width: 180,
+        height: 240,
+        padding: 12,
+        display: 'flex',
+        flexDirection: 'column',
+        alignItems: 'center',
+        justifyContent: 'space-around',
+        borderRadius: 4,
+        backgroundColor: 'white',
+        border: '1px solid lightgrey',
+      }}
+    >
+      <img src={'/editor/icons/light/special/palm4.png'} alt='aesthetic palmtree' />
```

Comment:
I'm not sure but I think the url should be wrapped in `getPossiblyHashedURL`. @Rheeseyb can you help me out?

Reply:
Definitely should.

Reply:
I've just turned it into a proper Icon so it can benefit from the helper / dealing with the hashed URL